package.json

```diff
@@ -21,18 +21,18 @@
   },
   "engineStrict": true,
   "dependencies": {
-    "colors": "~1.4.0",
+    "@colors/colors": "~1.5.0",
     "node-emoji": "~1.11.0",
-    "winston": "~3.3.3"
+    "winston": "~3.6.0"
   },
   "devDependencies": {
     "c8": "~7.11.0",
-    "codecov": "~3.8.2",
+    "codecov": "~3.8.3",
     "cross-env": "~7.0.3",
-    "eslint": "~8.5.0",
-    "eslint-plugin-mocha": "~10.0.0",
-    "mocha": "~9.1.0",
-    "standard": "~16.0.3"
+    "eslint": "~8.9.0",
```

Comment:
I still am puzzled why this project needs ESlint and Standard, but that's a discussion for a different day

Reply:
So I did some digging and eslint is being used specifically for linting the unit tests alongside [eslint-plugin-mocha](https://www.npmjs.com/package/eslint-plugin-mocha)